Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:latest
+
+MAINTAINER Stéphane Jeandeaux <stephane.jeandeaux@gmail.com>
+
+ENV MAVEN_VERSION=3.3.3 MAVEN_HOME=/usr/local/maven \
+    SBT_VERSION=0.13.8 SBT_HOME=/usr/local/sbt \
+    NODEJS_VERSION=0.12.7 NODEJS_HOME=/usr/local/node 
+
+ENV PATH=${PATH}:${MAVEN_HOME}/bin:${SBT_HOME}/bin:${NODEJS_HOME}/bin
+
+#bzip2 for npm (tar .bz2)
+RUN yum -y update && yum install -y git gcc-c++ make bzip2 
+
+RUN curl http://download.oracle.com/otn-pub/java/jdk/7u80-b15/jdk-7u80-linux-x64.rpm -L -C - -b "oraclelicense=accept-securebackup-cookie" -o /tmp/jdk.rpm && \
+    rpm -ivh /tmp/jdk.rpm && rm -f /tmp/jdk.rpm
+
+RUN curl -L http://wwwftp.ciril.fr/pub/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz -o /tmp/apache-maven.tar.gz && \
+    mkdir $MAVEN_HOME && tar --strip-components 1 -xzvf /tmp/apache-maven.tar.gz -C $MAVEN_HOME && \
+    rm -f /tmp/apache-maven.tar.gz
+
+RUN curl -sL "http://dl.bintray.com/sbt/native-packages/sbt/$SBT_VERSION/sbt-$SBT_VERSION.tgz" | gunzip | tar -x -C /usr/local 
+
+RUN curl -L https://nodejs.org/dist/v$NODEJS_VERSION/node-v$NODEJS_VERSION-linux-x64.tar.gz -o /tmp/nodejs.tar.gz && \
```

Comment:
Actually, on the build we do not depend on node being installed, because of awesome https://github.com/eirslett/frontend-maven-plugin so is there a reason to keep this one?